### ISSUE TITLE
update jasmine dependency version

### ISSUE THREAD

**Issue by DerZyklop** [mnasyrov/karma-jasmine-spec-tags#25]
```
npm WARN Could not resolve dependency:
npm WARN peer jasmine@"^3.3.0" from karma-jasmine-spec-tags@1.3.0
```

For me this package works fine with jasmine 4.4.0.

**Comment by archmoj** [comment]
+1

**Comment by mnasyrov** [comment]
Hi! I updated the peer dependencies. 

Fixed in karma-jasmine-spec-tags@2.0.0

**Comment by archmoj** [comment]
> Hi! I updated the peer dependencies.
> 
> Fixed in karma-jasmine-spec-tags@2.0.0

@mnasyrov Thank you very much! :pray:
